CI: odd failure of factory_source_tests

I cannot reproduce this locally and started occurring after `README.md` update. I assume something on travis-ci side, a dependency, or something else external to this repo is causing the issue.

```
======================================================================
FAIL: test_accept_request (tests.factory_source_tests.TestFactorySourceAccept)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py", line 165, in test_accept_request
    self.assertTrue(result['status'])
AssertionError: None is not true
-------------------- >> begin captured logging << --------------------
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: checking 770001
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Base:System/timezone@481ecbe0dfc63ece3a1f1b5598f7d96c -> openSUSE:13.2/timezone
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Checking in project openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking timezone in openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: new package
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking requests
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: ERROR: caught exception while checking openSUSE:Factory/timezone
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: timezone failed source submission check
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: 770001 ignored
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: checking 770001
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Base:System/timezone@481ecbe0dfc63ece3a1f1b5598f7d96c -> openSUSE:13.2/timezone
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Checking in project openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking timezone in openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: new package
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking requests
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: ERROR: caught exception while checking openSUSE:Factory/timezone
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: timezone failed source submission check
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: 770001 ignored
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: checking 770001
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Base:System/timezone@481ecbe0dfc63ece3a1f1b5598f7d96c -> openSUSE:13.2/timezone
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Checking in project openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking timezone in openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: new package
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking requests
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: ERROR: caught exception while checking openSUSE:Factory/timezone
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: timezone failed source submission check
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: 770001 ignored
--------------------- >> end captured logging << ---------------------
======================================================================
FAIL: test_source_not_in_factory (tests.factory_source_tests.TestFactorySourceAccept)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py", line 281, in test_source_not_in_factory
    self.assertTrue(result['factory_source_declined'])
AssertionError: None is not true
-------------------- >> begin captured logging << --------------------
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: checking 261411
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: home:lnussel:branches:openSUSE:Backports:SLE-12/plan@71e76daf2c2e9ddb0b9208f54a14f608 -> openSUSE:Backports:SLE-12/plan
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: Checking in project openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking plan in openSUSE:Factory
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: new package
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: DEBUG: checking requests
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: ERROR: caught exception while checking openSUSE:Factory/plan
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: plan failed source submission check
/home/travis/build/openSUSE/openSUSE-release-tools/tests/factory_source_tests.py: INFO: 261411 ignored
--------------------- >> end captured logging << ---------------------
```

--- a/tests/factory_source_tests.py
+++ b/tests/factory_source_tests.py
@@ -102,7 +102,7 @@ class TestFactorySourceAccept(unittest.TestCase):
                 </sourceinfo>
             """)
         httpretty.register_uri(httpretty.GET,
-            rr("/search/request?match=%28state%2F%40name%3D%27new%27+or+state%2F%40name%3D%27review%27%29+and+%28action%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27+or+submit%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27+or+action%2Fsource%2F%40project%3D%27openSUSE%3AFactory%27+or+submit%2Fsource%2F%40project%3D%27openSUSE%3AFactory%27%29+and+%28action%2Ftarget%2F%40package%3D%27timezone%27+or+submit%2Ftarget%2F%40package%3D%27timezone%27+or+action%2Fsource%2F%40package%3D%27timezone%27+or+submit%2Fsource%2F%40package%3D%27timezone%27%29+and+action%2F%40type%3D%27submit%27"),
+            rr("/search/request?match=%28state%2F%40name%3D%27new%27+or+state%2F%40name%3D%27review%27%29+and+%28action%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27+or+action%2Fsource%2F%40project%3D%27openSUSE%3AFactory%27%29+and+%28action%2Ftarget%2F%40package%3D%27timezone%27+or+action%2Fsource%2F%40package%3D%27timezone%27%29+and+action%2F%40type%3D%27submit%27"),
             match_querystring = True,
             responses = [
                 httpretty.Response( body = """
@@ -255,7 +255,7 @@ class TestFactorySourceAccept(unittest.TestCase):
             status = 404)
 
         httpretty.register_uri(httpretty.GET,
-            rr("/search/request?match=%28state%2F%40name%3D%27new%27+or+state%2F%40name%3D%27review%27%29+and+%28action%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27+or+submit%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27+or+action%2Fsource%2F%40project%3D%27openSUSE%3AFactory%27+or+submit%2Fsource%2F%40project%3D%27openSUSE%3AFactory%27%29+and+%28action%2Ftarget%2F%40package%3D%27plan%27+or+submit%2Ftarget%2F%40package%3D%27plan%27+or+action%2Fsource%2F%40package%3D%27plan%27+or+submit%2Fsource%2F%40package%3D%27plan%27%29+and+action%2F%40type%3D%27submit%27"),
+            rr("/search/request?match=%28state%2F%40name%3D%27new%27+or+state%2F%40name%3D%27review%27%29+and+%28action%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27+or+action%2Fsource%2F%40project%3D%27openSUSE%3AFactory%27%29+and+%28action%2Ftarget%2F%40package%3D%27plan%27+or+action%2Fsource%2F%40package%3D%27plan%27%29+and+action%2F%40type%3D%27submit%27"),
             match_querystring = True,
             body = """
                 <collection matches="0">

